.github/workflows/pytest.yml

```diff
@@ -115,8 +115,9 @@ jobs:
           mv .github/.coveragerc .
 
       - name: Test with pytest
+        id: pytest
         run: |
-          python3 -m pytest tests/${{matrix.test}} --color=yes --cov --cov-config=.coveragerc --durations=0 && exit_code=0|| exit_code=$?
+          python3 -m pytest tests/${{matrix.test}} --color=yes --cov --cov-config=.coveragerc --durations=0 -n auto && exit_code=0|| exit_code=$?
```

Comment:
out of curiosity, whta is the `-n` option? I couldn't find it on the pytest help message

Reply:
it tells xdist how many workers to use https://pytest-xdist.readthedocs.io/en/stable/